src/xlsx.js

```diff
@@ -0,0 +1,130 @@
+export class Workbook {
+  constructor(workbook) {
+    Object.defineProperties(this, {
+      _: {value: workbook},
+      sheetNames: {
+        value: workbook.worksheets.map((sheet) => sheet.name),
+        writable: false,
+        enumerable: true,
+      },
+    });
+  }
+  sheet(name, {range, headers = false} = {}) {
```

Comment:
Nit: Since these options aren’t used here, pass them through?

```suggestion
  sheet(name, options) {
```

Reply:
Hm, I felt since this is the public API boundary, we should be defensive here about the contents of options. At the same time, I lament the `f(...)` formatting this causes the inspector to fallback to and obscure the more useful signature in a notebook.